[2021-12-26 19:25:58,043] {taskinstance.py:1029} INFO - Dependencies not met for <TaskInstance: scraping_daft_wicklow.run_init_county scheduled__2021-12-25T11:10:00+00:00 [up_for_retry]>, dependency 'Not In Retry Period' FAILED: Task is not ready for retry yet but will be retried automatically. Current date is 2021-12-26T19:25:58.043526+00:00 and task will be retried at 2021-12-26T19:30:53.744790+00:00.
[2021-12-26 19:25:58,077] {local_task_job.py:99} INFO - Task is not able to be run
[2021-12-26 19:38:27,259] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_wicklow.run_init_county scheduled__2021-12-25T11:10:00+00:00 [queued]>
[2021-12-26 19:38:27,348] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_wicklow.run_init_county scheduled__2021-12-25T11:10:00+00:00 [queued]>
[2021-12-26 19:38:27,353] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-26 19:38:27,364] {taskinstance.py:1242} INFO - Starting attempt 2 of 4
[2021-12-26 19:38:27,366] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-26 19:38:27,436] {taskinstance.py:1262} INFO - Executing <Task(_PythonDecoratedOperator): run_init_county> on 2021-12-25 11:10:00+00:00
[2021-12-26 19:38:27,587] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_wicklow', 'run_init_county', 'scheduled__2021-12-25T11:10:00+00:00', '--job-id', '4728', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_wicklow.py', '--cfg-path', '/tmp/tmpeefagj2v', '--error-file', '/tmp/tmpcbclxcqy']
[2021-12-26 19:38:27,613] {standard_task_runner.py:77} INFO - Job 4728: Subtask run_init_county
[2021-12-26 19:38:27,485] {standard_task_runner.py:52} INFO - Started process 1146 to run task
[2021-12-26 19:38:28,215] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_wicklow.run_init_county scheduled__2021-12-25T11:10:00+00:00 [running]> on host 4391ebaded6b
[2021-12-26 19:38:28,873] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clement.liscoet@gmail.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_wicklow
AIRFLOW_CTX_TASK_ID=run_init_county
AIRFLOW_CTX_EXECUTION_DATE=2021-12-25T11:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-25T11:10:00+00:00
[2021-12-26 19:38:29,015] {python.py:152} INFO - Done. Returned value was: None
[2021-12-26 19:38:29,249] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=scraping_daft_wicklow, task_id=run_init_county, execution_date=20211225T111000, start_date=20211226T193827, end_date=20211226T193829
[2021-12-26 19:38:29,610] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-26 19:38:29,874] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
